packages/cli/src/cmds/preview/previewEnv.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+"use strict";
+
+import * as path from "path";
+import { Argv } from "yargs";
+import { FxError, ok, Result } from "@microsoft/teamsfx-api";
+import { environmentManager } from "@microsoft/teamsfx-core/build/core/environment";
+import * as constants from "./constants";
+import { YargsCommand } from "../../yargsCommand";
+
+// The new preview cmd `teamsfx preview --env ...`
+export default class PreviewEnv extends YargsCommand {
+  public readonly commandHead = `preview`;
+  public readonly command = `${this.commandHead}`;
+  public readonly description = "Preview the current application.";
+
+  public builder(yargs: Argv): Argv<any> {
+    yargs.option("folder", {
```

Comment:
Maybe you can use `yargs.options(xxx).options(xxx)`? There is also `RootFolderOptions` defined in the `constants.ts` file and you can use it.

Reply:
Thanks! Will update in next PR.